packages/cli/src/argparse.ts

```diff
@@ -345,7 +345,10 @@ export const CLI_ARGS = {
         '    $ export PAYMENT="bfeffdf57f29b0cc1fab9ea197bb1413da2561fe4b83e962c7f02fbbe2b1cd5401"\n' +
         '    $ stx call_contract_func SPBMRFRPPGCDE3F384WCJPK8PQJGZ8K9QKK7F59X contract_name' +
         '      contract_function 1 0 "$PAYMENT"\n' +
-        '    a9d387a925fb0ba7a725fb1e11f2c3f1647473699dd5a147c312e6453d233456\n' +
+        '     {\n' +
+        "       txid: '0x2e33ad647a9cedacb718ce247967dc705bc0c878db899fdba5eae2437c6fa1e1'," +
+        "       transaction: 'https://explorer.blockstack.org/txid/0x2e33ad647a9cedacb718ce247967dc705bc0c878db899fdba5eae2437c6fa1e1'" +
```

Comment:
why is this `explorer.blockstack.org` instead of `testnet-explorer.blockstack.org`?

Reply:
The example is for a mainnet transaction. The help examples don't currently check for the testnet/mainnet flag.